app/scripts/filters/resources.js

```diff
@@ -1351,8 +1316,21 @@ angular.module('openshiftConsole')
     };
   })
   .filter('hasAlternateBackends', function() {
-    return function(route) {
+    return function (route) {
       var alternateBackends = _.get(route, 'spec.alternateBackends', []);
       return !_.isEmpty(alternateBackends);
     };
+  })
+  // TODO: these two filters are defined in origin-web-catalog but are not available, need to figure out why or move to common
+  .filter('applicationHasDeployment', function ($filter) {
```

Comment:
OK I think we should look at it as a follow on to avoid having the same filters in multiple places.